build-tools/packages/build-cli/src/library/repoPolicyCheck/dockerfilePackages.ts

```diff
@@ -4,10 +4,11 @@
  */
 
 import fs from "node:fs";
+import path from "node:path";
 import { Handler, readFile, writeFile } from "./common.js";
 
 const serverPath = "server/routerlicious/";
-const serverDockerfilePath = `${serverPath}Dockerfile`;
+const serverDockerfilePath = (sPath: string): string => path.posix.join(sPath, "Dockerfile");
```

Comment:
How about bringing the extra `path.relative(process.cwd(), path.join(gitRoot, serverPath))` inside this function? Otherwise with 2 call sites and it being so simple I personally lean towards getting rid of it.